Core/src/Utilities/AnnealingUtility.cpp

```diff
@@ -8,14 +8,15 @@
 
 #include "Acts/Utilities/AnnealingUtility.hpp"
 
-/// @brief Gaussian function for weight calculation
+/// @brief Gaussian-like function for weight calculation
+/// Note: Factor 2 in denominator is included in inverse temperature
 ///
 /// @param chi2 Chi^2 value
-/// @param temp Temperature value
+/// @param invTemp Denominator 1/(2 * temperature)
 ///
-/// @return exp(-1./2. * chi2 / temp)
-static double gaussFunc(double chi2, double temp) {
-  return std::exp(-chi2 / (2. * temp));
+/// @return exp(-chi2 * invTemp)
+static double gaussFunc(double chi2, double invTemp) {
```

Comment:
The function name is a bit misleading. Is is not a Gaussian function since that would mean you have a quadratic term in the exponent, i.e. `exp(-x²)`. My suggestion would be to rename to represent its purpose instead of its content, e.g. `computeAnnealingWeight(...)`.

Reply:
Well, I have the quadratic term in the exponent. It is chi2.

Reply:
If you're happier if I change the name though, I will do it :)

Reply:
Also, renaming is absolutely not the point of this PR 🤷

Reply:
Renamed.